chore: bump cli version (#34)
ignite/apps: hermes/README.md

- Run both chain to be relayed.

> - add the hermes relayer app from remote:

Reviewer: Maybe "install" instead of "add":

```suggestion
- install the Hermes relayer app from a remote repository:
```

```shell
ignite app install -g github.com/ignite/apps/hermes
```

> - or clone the repo and add the hermes relayer app local:
> ```shell
> ignite app install -g $GOPATH/src/github.com/ignite/apps/hermes

Reviewer: Maybe something similar to this that uses "install" instead of "add":

```suggestion
- or clone the repository and install the Hermes relayer app from a local directory:
```shell
ignite app install -g PATH/TO/LOCAL/apps/hermes
```
